### ISSUE TITLE
Issue: same chatchannel names

### ISSUE THREAD

**Issue by salacryl** [RJacksonm1/node-dota2#304]
if a custom chat and a private chat shares the same name, you can't choose in which one you want to  send a message.
Maybe add type to sendMessage?

Greetings,
Björn

**Comment by Crazy-Duck** [comment]
Any reason why you're giving the same name to different chat channels?

**Comment by salacryl** [comment]
If the protocol allows it, do we really need to discuss why?

**Comment by Crazy-Duck** [comment]
Not really no, I was just curious ☺️ I'll see what I can do this week, probably as a second optional parameter.

**Comment by jimmydorry** [comment]
Understanding why you would ever want to do that, may motivate someone to implement it for you... otherwise putting it at the bottom of the queue. :|

**Comment by salacryl** [comment]
If you want to do so. Look: if it is possible by origin, OF COURSE people will do. I can't speak for all of them. This is how I test software. I don't change the purposes until they fit into the software, I change the software until it fits into the purpose. 
The user who want the bot has already his channels with >4000 chatusers. I guess it was easier to have them the same name.
It is the first time, that for a uninplemented (but - as I mentioned - legal by protocol) feature I'm asked "why?". It shouldn't matter.

**Comment by jimmydorry** [comment]
No it doesn't matter, but common courtesy is free too... and highly recommended when you are asking for people to spend their time doing something largely irrelevant to their own needs/usage.

**Comment by salacryl** [comment]
Why I have to repeat so often: if it a legal purpose to do so it isn't irrelevant. Ok I got it, it's your project and all users of this must provide they movivation beforehand.
Sounds like I'm on my own. Thank you for the help. Appreciated.

**Comment by jimmydorry** [comment]
It's straight forward to extend the library, and @Crazy-Duck even offered to do it later in the week if he gets time. 

I just think the way you were speaking was needlessly aggressive and mean-spirited... when the work done here is done freely and to help others as a consequence of our own labours. Perhaps it's a language barrier issue, but you are coming across as very rude, to me.

If you end up making something before @Crazy-Duck , feel free to pull-request it into this repo.

**Comment by salacryl** [comment]
Wait. You think I'm mean-spirited and also I'll do a pull request to share? This doesn't fit. No, I'm not mean spirit. I'm upset of how some people handle they projects. If you don't want to spend your time to fit in in peoples downstream projects, why even bother starting?
Look: I created a github account, I looked into the process of "how to fork and provide" to get an sharing process between us, because I believe that it is worth it and I have to face that I don't appreciate time you're spending and beeing accused to be mean-spirited? All this because I'm just straight forward.  
@Crazy-Duck, Thank you. I'll also look into. So don't rush it :) I'll keep you informed.
